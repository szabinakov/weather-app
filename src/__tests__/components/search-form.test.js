import React from 'react'
import SearchForm from '../../components/search-form'
import { render } from '@testing-library/react'

describe('Search-form', () => {

    it('matches the snapshot', () => {
        const { asFragment } = render (<SearchForm />)

        expect(asFragment).toMatchSnapshot()
    })

    it('renders the input', () => {
        const { getByTestId } = render (<SearchForm />)

        expect(getByTestId('inputId')).toHaveClass('inputClass')
    })

    it('renders the button', () =>{
        const { getByTestId } = render (<SearchForm />)

        expect(getByTestId('buttonId')).toHaveClass('buttonClass')
    })
})