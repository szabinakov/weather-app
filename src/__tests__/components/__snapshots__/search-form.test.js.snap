// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search-form matches the snapshot 1`] = `[Function]`;
